src/components/Card/card.module.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+    box-shadow: 0 0 15px -7px rgba(0, 0, 0, 0.65);
+    border-radius: 10px;
+    padding: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin-bottom: 2rem;
+    cursor: pointer;
+  }
+.card:hover {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  }
+.prTitle {
+    color: #041484;
+    font-size: 2rem;
+    font-weight: bold;
+  }
+.cardFooter {
+    display: flex;
+    flex-direction: row;
```

Comment:
NIT: By default `display: flex` takes the direction as `row`
So no need to specify.

Reply:
Deleted flex-direction: row

https://github.com/Real-Dev-Squad/website-status/pull/101/commits/f84f9afa7df2679012784eb4f1b1558a2496fdb4